packages/expo-image-picker/android/build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
 
   repositories {
     mavenCentral()
+    maven { url 'https://jitpack.io' }
```

Comment:
that's `buildscript.repositories` which may not what you want?
for app to download the dependency, we already have jitpack in our template:
https://github.com/expo/expo/blob/594bb7e16883bf345e4bf14c3e6380e7df176c79/templates/expo-template-bare-minimum/android/build.gradle#L39

Reply:
Oh, true 🤦 
What would happen if somebody is not having `jitpack.io` in their main gradle file?
Isn't is better for us to have each package be self-describing and storing it's own independent configuration? 🤔 I follow the idea of `package`-centric approach where each package-related aspect is pushed down to the package itself, just to enable it's usage standalone-ly.